misc/services/acrn_manager/acrn_vm_ops.c

```diff
@@ -387,24 +387,6 @@ int stop_vm(const char *vmname, int force)
 	return ack.data.err;
 }
 
-int suspend_vm(const char *vmname)
-{
-	struct mngr_msg req;
-	struct mngr_msg ack;
-
-	req.magic = MNGR_MSG_MAGIC;
-	req.msgid = DM_SUSPEND;
-	req.timestamp = time(NULL);
-
-	send_msg(vmname, &req, &ack);
-
-	if (ack.data.err) {
-		printf("Unable to suspend vm. errno(%d)\n", ack.data.err);
-	}
-
-	return ack.data.err;
-}
-
 int resume_vm(const char *vmname, unsigned reason)
```

Comment:
What is the reason you are not removing this function (while you are removing the `suspend_vm` one?

Reply:
@gvancuts Thanks for your comments. resume_vm is also used by acrnd, if ACRND_RESUME command is useless, I think we also can remove resume_vm in another patch. BTW, in this patch, only focus on removing acrnctl suspend and acrnctl resume.